### DISCUSSION TITLE
"No Data" shown and getting "NetworkError when attempting to fetch resource." in Grafana.

### DISCUSSION THREAD

**Discussion by mikec-design** [teslamate-org/teslamate#4013] [Q&A]
Hi everyone. Up until a few days ago everything was working perfectly with Teslamate. Now when I try to load any dashboards I get a "NetworkError when attempting to fetch resource." error appears and all the graphs will show no data. 

I've restarted the Raspberry Pi, stopped and started the container, but nothing is working.

The output from sudo docker ps -a shows that postgres:15's status is "Restarting (127) 25 seconds ago". Then it was up for 5 seconds before restarting again. All the other images have no issues.

**Comment by guicane** [comment]
I'm having the same issue and can't see anything in the logs...

**Comment by brianmay** [comment]
I have been getting "NetworkError when attempting to fetch resource." from Grafana lately. Today it is "Unexpected error".

But the data loads fine.

The network tab shows that there is a request to "/frontend-metrics" that is blocked by uBlock Origin. This issue does not happen under Brave, only Firefox. LATER: I disabled uBlock Origin, had to also disable Ghostery, Now the error has gone

BUT: There are some websites that don't display correctly on my Firefox. Only two websites are affected - grafana and Google sheets. With Grafana I need to mouse over the graphs to see the details. Again this doesn't happen with Brave. My theory is this is some bad combination of proprietary nvidia drivers, Firefox, and Linux. This problem seems to be very distinct to the first issue.

Not sure if any of this is helpful or not...

**Reply by guicane** [reply]
Good troubleshooting there. I can confirm I am also using Firefox with uBlock Origin when I see the error. Have disabled it now in Grafana and it seems to be fine since.

**Comment by mikec-design** [comment]
I'm also running Firefox with Ublock Origin and a Pihole. Checking if the Pihole was the problem, I've disabled it and tried Chrome, but the restarting Postgres database remains. It's restarting every 16-17 seconds.

Any suggestions on how to get the database stable, or how I can extract the data?

**Reply by cwanja** [reply]
Have you tried disabling Ublock Origin? Several threads point to this as an issue. Although I struggle to see how that would cause your database from restarting. 

What about available storage on your device?

Personally, running a Pi-hole without any issues with Grafana.

**Reply by mikec-design** [reply]
Yes, Tried logging in with Chrome and Edge, but I end up with N/A or No data displayed in the charts.

Looks like I have lots of room left on the SD Card.

`Filesystem     1M-blocks  Used Available Use% Mounted on`
`/dev/root         28869M 7001M    20639M  26% /`
`devtmpfs            333M    0M      333M   0% /dev`
`tmpfs               461M    0M      461M   0% /dev/shm`
`tmpfs               185M    2M      184M   1% /run`
`tmpfs                 5M    1M        5M   1% /run/lock`
`/dev/mmcblk0p1      255M   51M      205M  20% /boot`
`tmpfs                93M    1M       93M   1% /run/user/1000`
